modules/microshift-manifests-deletion-overview.adoc

```diff
@@ -8,11 +8,11 @@
 
 By default, {microshift-short} searches for deletion manifests in the `delete` subdirectories within the manifests path. When a user places a manifest in these subdirectories, {microshift-short} removes the manifests when the system is started. Read through the following to understand how manifests deletion works in {microshift-short}.
 
-. Each time the system starts, before applying the manifests, {microshift-short} scans the following `delete` subdirectories within the configured manifests directory to identify the Kustomize manifests that need to be deleted:
+. Each time the system starts, before applying the manifests, {microshift-short} scans the following `delete` subdirectories within the configured manifests directory to identify the manifests that need to be deleted:
 
 * /usr/lib/microshift/manifests/delete
 * /usr/lib/microshift/manifests.d/delete/*
 * /etc/microshift/manifests/delete
 * /etc/microshift/manifests.d/delete/*
```

Comment:
Out of scope to your work, so up to you, but per the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference), filepaths are some of the elements that always need to be in monospace:
```suggestion
* `/usr/lib/microshift/manifests/delete`
* `/usr/lib/microshift/manifests.d/delete/*`
* `/etc/microshift/manifests/delete`
* `/etc/microshift/manifests.d/delete/*`
```

Reply:
done